virttest/test_setup.py

```diff
@@ -603,11 +632,9 @@ def set_node_num_huge_pages(self, num, node, pagesize):
         """
         node_page_path = "%s/node%s" % (self.sys_node_path, node)
         node_page_path += "/hugepages/hugepages-%skB/nr_hugepages" % pagesize
-        if not os.path.isfile(node_page_path):
-            raise ValueError("%s page size nr_hugepages file of node %s did "
-                             "not exist" % (pagesize, node))
-        process.system("echo %s > %s" % (num, node_page_path), shell=True)
-        if int(num) != self.get_node_num_huge_pages(node, pagesize):
+        obj = kernel_interface.SysFS(node_page_path)
+        obj.sys_fs_value = num
+        if obj.sys_fs_value != int(num):
```

Comment:
`num` should always be int.  Would you please correct the parameter description as well, for both set_ and get_ function, thanks.

Reply:
Due to some num maybe get a str from the config, and the coder forget to turn into int.